processor/src/main/java/org/mapstruct/ap/internal/util/Message.java

```diff
@@ -92,6 +92,7 @@ public enum Message {
     PROPERTYMAPPING_CANNOT_DETERMINE_SOURCE_PARAMETER_FROM_TARGET("No property named \"%s\" exists in source parameter(s). Please define the source explicitly."),
     PROPERTYMAPPING_NO_SUITABLE_COLLECTION_OR_MAP_CONSTRUCTOR( "%s does not have an accessible copy or no-args constructor." ),
     PROPERTYMAPPING_EXPRESSION_AND_CONDITION_QUALIFIED_BY_NAME_BOTH_DEFINED( "Expression and condition qualified by name are both defined in @Mapping, either define an expression or a condition qualified by name." ),
+    PROPERTYMAPPING_TARGET_HAS_NO_TARGET_PROPERTIES( "Target \"%s\" has no target properties, targeted by this mapping method: \"%s\"", Diagnostic.Kind.WARNING ),
```

Comment:
Do we really need the mapping method here? The line on which we report the warning is the method itself so it should not be needed to also put the method here.